src/management/__generated/managers/ActionsManager.ts

```diff
@@ -0,0 +1,810 @@
+  }
+
+  /**
+   * Retrieve an action by its ID.<br/>
+   * Get an action
+   */
+  async get(
+    requestParameters: GetActionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<GetActions200ResponseActionsInner> {
+    const response = await this.getRaw(requestParameters, initOverrides);
+    return await response.value();
+  }
+
+  /**
+   * Retrieve a specific version of an action. An action version is created whenever<br/>an action is deployed. An action version is immutable, once created.<br/>
+   * Get a specific version of an action
+   * @throws {RequiredError}
+   * @memberof ActionsManager
+   */
+  async getVersionRaw(
+    requestParameters: GetActionVersionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<runtime.ApiResponse<GetActionVersions200ResponseVersionsInner>> {
+    if (requestParameters.actionId === null || requestParameters.actionId === undefined) {
+      throw new runtime.RequiredError(
+        'actionId',
+        'Required parameter requestParameters.actionId was null or undefined when calling getVersion.'
+      );
+    }
+    if (requestParameters.id === null || requestParameters.id === undefined) {
+      throw new runtime.RequiredError(
+        'id',
+        'Required parameter requestParameters.id was null or undefined when calling getVersion.'
+      );
+    }
+
+    const response = await this.request(
+      {
+        path: `/actions/actions/{actionId}/versions/{id}`
+          .replace(`{${'actionId'}}`, encodeURIComponent(String(requestParameters.actionId)))
+          .replace(`{${'id'}}`, encodeURIComponent(String(requestParameters.id))),
+        method: 'GET',
+      },
+      initOverrides
+    );
+
+    return new runtime.JSONApiResponse(response);
+  }
+
+  /**
+   * Retrieve a specific version of an action. An action version is created whenever<br/>an action is deployed. An action version is immutable, once created.<br/>
+   * Get a specific version of an action
+   */
+  async getVersion(
+    requestParameters: GetActionVersionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<GetActionVersions200ResponseVersionsInner> {
+    const response = await this.getVersionRaw(requestParameters, initOverrides);
+    return await response.value();
+  }
+
+  /**
+   * Retrieve all of an action\'s versions. An action version is created whenever<br/>an action is deployed. An action version is immutable, once created.<br/>
+   * Get an action\'s versions
+   * @throws {RequiredError}
+   * @memberof ActionsManager
+   */
+  async getVersionsRaw(
+    requestParameters: GetActionVersionsRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<runtime.ApiResponse<GetActionVersions200Response>> {
+    if (requestParameters.actionId === null || requestParameters.actionId === undefined) {
+      throw new runtime.RequiredError(
```

Comment:
(probably not doable but) Would be nice to import `import { RequiredError } from '../../runtime';` so we can do `throw new RequiredError(`